utils/forest_shim/src/clock.rs

```diff
@@ -0,0 +1,5 @@
+// Copyright 2019-2023 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use fil_actors_runtime_v9::builtin::network;
+pub const EPOCHS_IN_DAY: fvm_shared::clock::ChainEpoch = network::EPOCHS_IN_DAY;
```

Comment:
Actually, let's hoist out the constant and define it ourselves.

Reply:
okay